src/Perl6/Actions.nqp

```diff
@@ -9688,6 +9709,7 @@ class Perl6::Actions is HLL::Actions does STDActions {
             pos_slurpy        => $pos_slurpy,
             named_slurpy      => $named_slurpy,
             placeholder       => $full_name,
+            ast               => $/,
```

Comment:
Is this a common name used for this purpose?

All I ever seen `ast` used for made stuff, like QAST trees, not for Perl6::Grammar objects. `node` is used for those usually in things like QAST nodes.

Reply:
Thanks for the suggestion! I'll incorporate that.